src/main/scala/com/typesafe/sbt/packager/linux/LinuxPlugin.scala

```diff
@@ -249,4 +249,7 @@ object LinuxPlugin extends AutoPlugin {
     )
   }
 
+  final def chdir(installLocation: String, packageName: String): String =
+    s"$installLocation/$packageName"
+
```

Comment:
I see little value in the addition of this function. It is another level of indirection and doesn't actually change the dir. :-)